Simplified.xcodeproj/project.pbxproj

```diff
@@ -1241,7 +1241,7 @@
 			attributes = {
 				CLASSPREFIX = NYPL;
 				LastSwiftUpdateCheck = 0730;
-				LastUpgradeCheck = 0820;
+				LastUpgradeCheck = 0920;
```

Comment:
What is this attribute referring to?

Reply:
I went ahead and let Xcode verify that the project is up to its current standards. This involved Xcode enabling some new warnings and setting `LastUpgradeCheck` to indicate that Xcode 9.2 had done its job. See 541c07489ef35ed16122e8b7658f48fde3d033b3 for the exact change.